recipe/build.sh

```diff
@@ -1,11 +1,28 @@
 #!/usr/bin/bash
 
-set -x
+set -ex
 
-cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* ./config
+cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* ./build-aux
+
+if [[ "$target_platform" == linux* ]]; then
+  rm -f tests/scripts/functions/wildcard
+elif [[ "$target_platform" == osx* ]]; then
+  rm -f tests/scripts/variables/INCLUDE_DIRS
+fi
+
+if [[ ${target_platform} =~ linux* || ${target_platform} =~ .*aarch64.* || ${target_platform} =~ .*ppc64le.* ]]; then
+  config_opts="--disable-dependency-tracking"
+else
+  config_opts=""
+fi
 
 ./configure --prefix=${PREFIX} \
-            --host=${HOST}
-make -j${CPU_COUNT} ${VERBOSE_AT}
-make check
-make install
+            --host=${HOST} \
+            ${config_opts}
+# make -j${CPU_COUNT} ${VERBOSE_AT}
+
+# bootstrap building make without make
+bash build.sh
```

Comment:
isn't this an endless recusion?!?

Reply:
Do we need **bash** in `req/build` as c-f https://github.com/conda-forge/make-feedstock/commit/bdcc1216ff252ec0f29605ecb1d725d2b8e05a5b ?